### PR TITLE
Favicons added

### PR DESCRIPTION
**Pull request by Rasosm**

### CODE REVIEW

**Review by EvgenySivaykin — DISMISSED** [review]
OK

File: index.html [hunk 1]
```diff
@@ -8,6 +8,15 @@
     <title>Braintech | IT Solutions and Technology Startup HTML Template</title>
```
Comment (liepsneletech) [inline index.html:8]: Not about favicons :). In the title shouldn't we remove the text "HTML Template"?

**Review by liepsneletech — CHANGES_REQUESTED** [review]
Small fix : )

File: index.html (hunk 1 above)
Comment (zilva149) [reply index.html:8]: yes, original has it, but we should change it

**Review by zilva149 — APPROVED** [review]
Looks good

**Pull request merged**